### DISCUSSION TITLE
If a product entry has been made to a table, can you open the table without refreshing the page in the Table management section.

### DISCUSSION THREAD

**Discussion by lim10soft** [Blair2004/NexoPOS#763] [Gastro 4.x Ideas & Issues]
Can You Make the Table Management Section Automatically Refresh When the Table Is Opened ?
![2022-05-09_13-52-28](https://user-images.githubusercontent.com/102427740/167395963-95368619-4ef4-4997-8b38-c821ced5438b.png)

**Comment by Blair2004** [comment]
The system already have a infinite query to the serve for fetching ready meals. We're afraid adding another infinite request here might overload the server and globally make the app slow. We're studying the path of sockets.

**Reply by lim10soft** [reply]
If only the table where the order is entered is renewed instead of constantly being renewed, this will not be a problem. Can you do it this way, sir?
